Contact me section

abdulrahmanshr75 opened the pull request:
- added the contact me section
- added the form
- limited the name and text area
- implemented the button interactions
- added the illustration

abdulrahmanshr75 committed:
update

abdulrahmanshr75 committed:
Update Styles

aeon9300 committed:
Add float style to social icons

abdulrahmanshr75 committed:
add-contact-section

aeon9300 committed:
Modify height of contact section

abdulrahmanshr75 committed:
modify the shape and button

aeon9300 committed:
Restrict number of characters of the Name and Text in contact form

abdulrahmanshr75 committed:
added formspree endpoint

abdulrahmanshr75 committed:
added formspree endpoint

abdulrahmanshr75 committed:
fixed linter errors

abdulrahmanshr75 committed:
update screenshot in readme

andersonlebon commented on index.html:
```diff
@@ -150,65 +152,134 @@ <h2>Multi-Post Stories Gain+Glory</h2>
+            maxlength="500"
+            placeholder="Enter Text Here"
+            required
+          >
```
- [ ] Kindly remove this unnecessary character `>`

andersonlebon commented on index.html:
```diff
@@ -150,65 +152,134 @@ <h2>Multi-Post Stories Gain+Glory</h2>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form
+          class="contact-form"
+          action="https://formspree.io/f/mlezeeov"
+          method="post"
+        >
+          <input
+            class="Form-Name"
+            name="name"
+            type="text"
+            maxlength="30"
+            placeholder="Full Name"
+            required
+          />
+          <input
+            class="Form-Email"
+            name="Email"
+            type="email"
+            placeholder="Email Address"
+            required
+          />
+          <textarea
+            class="Form-text"
+            name="text"
+            maxlength="500"
+            placeholder="Enter Text Here"
+            required
```
- [ ] One of the requirements says that you should stick to the design templates in Figma as much as possible, this includes the font, colors, images, and spacing (margin/padding). Kindly let's go ahead a  make our form look exactly like the template one. Refer yourself to these screenshots below  :point_down: :point_down: 

 Your design | The template
-----------------|------------------
![Screenshot from 2021-12-07 18-44-10](https://user-images.githubusercontent.com/65068771/145071385-21476aad-2904-404b-92c9-993d4f4484f3.png) | ![Screenshot from 2021-12-07 18-40-54](https://user-images.githubusercontent.com/65068771/145071332-eb2b6e01-0ac6-43b5-bd5f-82d7cf95347c.png)

andersonlebon reviewed (changes requested):
### Hi @abdulrahmanshr75  

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

![almost-there](https://user-images.githubusercontent.com/65068771/141811112-252360b9-757f-4279-bb3b-b7a2a4675ad3.gif)
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

abdulrahmanshr75 committed:
Changes

- removed the unnecessary character.
- redesigned the template so it can be as the figma design.

abdulrahmanshr75 committed:
Fixed styles linter errors

alonsofalconi reviewed (approved):
Hi @abdulrahmanshr75 and @aeon9300,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

![](https://media1.giphy.com/media/d31w24psGYeekCZy/giphy.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
